doc/api/addons.md

```diff
@@ -155,12 +155,14 @@ they were created.
 The context-aware addon can be structured to avoid global static data by
 performing the following steps:
 * Define a class which will hold per-addon-instance data and which has a static
-member of the form
-  ```cpp
+member of the form.
+
+```cpp
   static void DeleteInstance(void* data) {
     // Cast `data` to an instance of the class and delete it.
   }
```

Comment:
```suggestion
static void DeleteInstance(void* data) {
  // Cast `data` to an instance of the class and delete it.
}
```

Reply:
sorry for some reasons my browser was showing this:

![Screenshot (61)](https://user-images.githubusercontent.com/45990659/89711255-5e65a000-d99e-11ea-9056-16c4af210068.png)

but now it's fine.